src/spatio_temporal_voxel_layer.cpp

```diff
@@ -175,6 +176,8 @@ void SpatioTemporalVoxelLayer::onInitialize(void)
     source_node.param("decay_acceleration", decay_acceleration, 0.);
     // performs an approximate voxel filter over the data to reduce
     source_node.param("voxel_filter", voxel_filter, false);
+    // Skips built-in pcl filtering altogether (BYO filtering)
```

Comment:
Skips? I think this should be describing what the param is / does. 

Please also add this param to the param description in the readme

Reply:
Good catch. An early implementation was negative, i.e., skip_filtering or something like that.